packages/chaire-lib-common/src/services/routing/OSRMRoutingService.ts

```diff
@@ -88,6 +88,7 @@ export default class OSRMRoutingService extends RoutingService.default {
             const origin = waypoints[0];
             const destination = waypoints[waypoints.length - 1];
 
+            // TODO: Don't use osrmRouting params in chaire-lib-common
```

Comment:
Here the TODO should be more Determine if this value should be a config or a preference